src/main/resources/db/migration/V14__create_spm_svar_tables.sql

```diff
@@ -0,0 +1,16 @@
+CREATE TABLE sporsmal
+(
+    id      VARCHAR(64)   NOT NULL PRIMARY KEY,
+    tekst   VARCHAR(255)  NOT NULL
```

Comment:
Burde vi hatt et egent felt for shortName kanskje? for eksempel 
id 1
shortName: FORSIKRING
tekst: Har du forsikring?

blir litt lettere når man skal hente ut svar kanskje, da man kan sjekke på shortName istedet for hele teksten?

Reply:
Ja, men hvordan blir det hvis vi endrer på spørsmålene..?

Reply:
Tror kanskje det er best å beholde hele sporsmålet, for sporbarhet.
Mens få å hente det ut og ha statestikk, er det letter med shortName

Reply:
Ja spørsmålet skal vi beholdet, jeg tenker bare på å ha shortName eller noe i tillegg

Reply:
Da kan kanskje shortname være et enum i koden som har verdiene FORSIKRING, ARBEIDSSITUASJON, FRAVAER og FRAVAERSPERIODE?